pkg/kubelet/cm/node_container_manager_test.go

```diff
@@ -297,9 +297,74 @@ func TestNodeAllocatableInputValidation(t *testing.T) {
 			NodeConfig: nc,
 			capacity:   tc.capacity,
 		}
-		if err := cm.validateNodeAllocatable(); err != nil && !tc.invalidConfiguration {
```

Comment:
I think the result check logic should be modified too. If `validateNodeAllocatable` returns no error, but we set the configuration to `invalid`, it should also log error